xt/testapp/t/003_test_idps_configuration.t

```diff
@@ -0,0 +1,63 @@
+use strict;
```

Comment:
This block can be replaced with `use Test::Lib; use Test::Net::SAML2;`

Reply:
perl -I ../../blib/lib/ -I blib/lib/ t/003_test_idps_configuration.t
unable to find t/lib directory in /home/tim/perl-Net-SAML2/xt/testapp/t/ at /home/tim/perl5/lib/perl5/Test/Lib.pm line 32.
BEGIN failed--compilation aborted at t/003_test_idps_configuration.t line 1.

It appears relative to the current directory.  So I would need to copy a bunch of code to xt/testapp/t